src/Scripting/CoreTestUtilities/ObjectFormatterFixtures/MockDesktopSpinLock.cs

```diff
@@ -17,9 +17,7 @@ namespace ObjectFormatterFixtures
     [DebuggerDisplay("IsHeld = {IsHeld}")]
     internal struct MockDesktopSpinLock
     {
-#pragma warning disable IDE0044 // Add readonly modifier - See https://github.com/dotnet/roslyn/issues/47225
-        private volatile int m_owner;
-#pragma warning restore IDE0044 // Add readonly modifier
+        private readonly int m_owner;
```

Comment:
It needs a careful review.
I removed volatile assuming that this can't be modified by multiple threads. If that is not correct, we may need to update the analyzer.

Reply:
> It needs a careful review.
> I removed volatile assuming that this can't be modified by multiple threads. If that is not correct, we may need to update the analyzer.

@mavasani, As I'm not 100% sure of the change. So pinging you on this in case this comment was unnoticed.